docs/en/week12/12-3.md

```diff
@@ -0,0 +1,239 @@
+---
+lang-ref: ch.12-3
+title: MPC (EBM version)
+lecturer: Alfredo Canziani
+authors: Yang Zhou, Daniel Yao
+date: 28 Apr 2021
+---
+
+
+## Action plan
+- Model predictive control **[Here we are today]**
+    - Backprop through kinematic equation
+    - Minimisation of the latent
+- Truck backer-upper
+    - Learning an emulator of the kinematics from observations
+    - Training a policy
+- PPUU
+    - Stochastic environment
+    - Uncertainty minimisation
+    - Latent decoupling
+
+## State transition equations -- Evolution of the state
+
+Here we discuss a state transition equation where $x$ represents the state, $u$ represents control. We can formulate the state transition function in a continuous-time system where $x(t)$ is a function of continuous variable $t$.
+$\dot{x} = f(x,u)$
```

Comment:
Vectors use `\vect{}`, which calls `\bm{}`.
Also, please use `$$` and add an empty line before.
Finally, have both equations in a `\begin{align}`/`\end{align}` environment.

Reply:
I don't think we can use align environment, I checked the repo from last year, there was no example about align environment as well.

Reply:
https://katex.org/docs/supported.html#environments

Reply:
Also, `x` and `u` are still not bolded here.

Reply:
Oh I found out the wan. align does not work, aligned works. I will submit a PR addressing all issues.

Reply:
A PR for what? I haven't merged this yet. I'm confused.